ui/src/models/versions.js

```diff
@@ -0,0 +1,12 @@
+export default class Version {
+  constructor (json = {}) {
+    this.id = json.id;
+    this.versionNum = json.version;
+    this.appId = json.appId;
+    this.isDisabled = json.disabled;
+    this.disabledMessage = json.disabledMessage;
+    this.numOfAppLaunches = json.numOfAppLaunches;
+    this.currentInstalls = json.numOfCurrentInstalls;
+    this.lastLaunchedAt = json.lastLaunchedAt;
```

Comment:
If using a model should we use the default values we currently do for some of these fields?